src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
```

Comment:
Is it OK to load the `collectors.xml` file unconditionally? Is it really used when `debug = false` ? If this can be changed, the solution would be trivial: move `$loader->load('collectors.xml');` under the `if ($container->getParameter('kernel.debug'))` condition.